README.md

```diff
@@ -420,11 +421,105 @@ description: "This high priority class should be used for MarkLogic pods only."
 
 2. Set priorityClassName in the values.yaml file or using the --set flag while installing the chart. The value of priorityClassName should be set to one of the added PriorityClassName.
 
-
 ## Notice
 
 To use transactional functionality with MarkLogic, you have to set up Ingress and configure cookie-based session affinity. This function will be supported in a future release.
 
+# Upgrades
+
+MarkLogic Kubernetes Helm chart will be released in MAJOR, MINOR or PATCH form. Major releases may include breaking changes and new features that require configuration changes in values.yaml file. So it's always recommended to go through the changes in the release and test the upgrade on a non production environment. Minor and Patch release will inlude bug fixes and non-breaking changes. 
+
+Before you begin, following are some recommendations for upgrade procedures:
+* Make sure to have the latest version of Helm installed on your machine.
+* Avoid using --reuse-values option with helm upgrade to ensure the changes in new values.yaml from updated chart is merged into your release.
+* Always use values.yaml file using `-f` option and avoid using --set option while installing and upgrading chart to ensure your release has all new values.
+* The bootstrap host in the MarkLogic statefulset needs to be upgraded before any other node in the cluster as it has the security and schemas database. So, we suggest using OnDelete upgrade strategy instead of RollingUpgrade that starts upgrading pod from the largest ordinal to the smallest. For more information on upgrade strategies, please refer https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#update-strategies
+* It's important to have database backup for recovery in case of upgrade failure. Please refer https://docs.marklogic.com/guide/admin/backup_restore for detailed procedure on backing up and restoring a database.
+
+This section describes three upgrade procedures:
+1. [Upgrading MarkLogic Kubernetes Helm Chart version](#upgrading-markLogic-version-in-your-release)
+2. [Upgrading MarkLogic version in your release](#upgrading-markLogic-version-in-your-release)
+3. [Upgrading MarkLogic version and MarkLogic Kubernetes Helm Chart version at once](#upgrading-markLogic-version-and-markLogic-kubernetes-helm-chart-version-at-once)
+
+## Upgrading MarkLogic Kubernetes Helm Chart Version
+
+This upgrade is required whenever there is a requirement at your end or when MarkLogic Kubernetes Helm Chart releases a new version with new features or bug fixes. The general procedure to upgrade MarkLogic Kubernetes Helm Chart is as follows:
+
+1. Update the chart repository using the below command to get the new version of the chart
+```
+helm repo update
+```
+2. Check the available upgrades of MarkLogic Kuberenetes Helm chart using following command,
+```
+helm search repo marklogic/marklogic
```

Comment:
it should be `helm search repo marklogic`

Reply:
updated chart name